backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/entities/Token.java

```diff
@@ -0,0 +1,24 @@
+package com.google.gpay.virtualqueue.backendservice.entities;
+
+import lombok.Data;
+
+@Data
+public class Token {
+    private Integer shopId;
+    private Integer tokenNo;
+
+    enum Status {
+        STATUS_UNKNOWN, ACTIVE, EXPIRED, CANCELLED_BY_CUSTOMER, CANCELLED_BY_SHOP_KEEPER;
```

Comment:
Let's use the term shop_owner everywhere instead of using both shop_keeper and show_owner interchangeably.